persistence/provider/memory/eventstore.go

```diff
@@ -19,7 +20,11 @@ func (t *transaction) SaveEvent(
 		return 0, err
 	}
 
-	return t.event.stageSave(&t.ds.db.event, env), nil
+	t.result.EventItems = append(
+		t.result.EventItems,
+	)
+
```

Comment:
Probably a remnant of an earlier approach.

```suggestion
	t.result.EventItems = append(
		t.result.EventItems,
	)

```

Reply:
Hmm, is it one of those suggestions that are the same as the original, GH would not probably like that.

Reply:
Sorry man! Fixed it. I think there was an extra newline after or something too.